src/main/java/com/auth0/json/mgmt/Scope.java

```diff
@@ -0,0 +1,35 @@
+package com.auth0.json.mgmt;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Scope {
+    @JsonProperty("description")
+    private String description;
+    @JsonProperty("value")
+    private String value;
+
+    @JsonCreator
+    public Scope(@JsonProperty("value") String value,
```

Comment:
Value is the only required property. Let's have a single constructor with value instead of both.

Reply:
done